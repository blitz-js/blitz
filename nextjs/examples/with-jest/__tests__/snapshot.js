import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../pages/index'

it('renders homepage unchanged', () => {
  const tree = renderer.create(<Index />).toJSON()
  expect(tree).toMatchSnapshot()
})
