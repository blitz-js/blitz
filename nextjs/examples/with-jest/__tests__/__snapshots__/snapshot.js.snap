// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="container"
>
  <main>
    <h1
      className="title"
    >
      Welcome to
      <a
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
    <p
      className="description"
    >
      Get started by editing
      <code>
        pages/index.js
      </code>
    </p>
    <div
      className="grid"
    >
      <a
        className="card"
        href="https://nextjs.org/docs"
      >
        <h3>
          Documentation →
        </h3>
        <p>
          Find in-depth information about Next.js features and API.
        </p>
      </a>
      <a
        className="card"
        href="https://nextjs.org/learn"
      >
        <h3>
          Learn →
        </h3>
        <p>
          Learn about Next.js in an interactive course with quizzes!
        </p>
      </a>
      <a
        className="card"
        href="https://github.com/vercel/next.js/tree/master/examples"
      >
        <h3>
          Examples →
        </h3>
        <p>
          Discover and deploy boilerplate example Next.js projects.
        </p>
      </a>
      <a
        className="card"
        href="https://vercel.com/new"
      >
        <h3>
          Deploy →
        </h3>
        <p>
          Instantly deploy your Next.js site to a public URL with Vercel.
        </p>
      </a>
    </div>
  </main>
</div>
`;
